chart/templates/server-ide-configmap.yaml

```diff
@@ -46,33 +46,22 @@ options:
     orderKey: "00"
     title: "VS Code"
     type: "browser"
+    label: "Browser"
     logo: "https://ide.{{ $.Values.hostname }}/image/ide-logo/vscode.svg"
+    # latestLogo: "https://ide.{{ $.Values.hostname }}/image/ide-logo/vscodeInsiders.svg"
     image: {{ (include "stable-image-full" (dict "root" $ "gp" $gp "comp" $gp.components.workspace.codeImage)) }}
-  code-latest:
-    orderKey: "01"
-    title: "VS Code"
-    type: "browser"
-    logo: "https://ide.{{ $.Values.hostname }}/image/ide-logo/vscodeInsiders.svg"
-    tooltip: "Early access version, still subject to testing."
-    label: "Insiders"
-    image: {{ (include "insider-image-full" (dict "root" $ "gp" $gp "comp" $gp.components.workspace.codeImage)) }}
-    resolveImageDigest: true
+    latestImage: {{ (include "insider-image-full" (dict "root" $ "gp" $gp "comp" $gp.components.workspace.codeImage)) }}
 
   # Desktop IDEs
   code-desktop:
     orderKey: "02"
     title: "VS Code"
     type: "desktop"
+    label: "Desktop"
     logo: "https://ide.{{ $.Values.hostname }}/image/ide-logo/vscode.svg"
+    # latestLogo: "https://ide.{{ $.Values.hostname }}/image/ide-logo/vscodeInsiders.svg"
     image: {{ (include "gitpod.comp.imageFull" (dict "root" $ "gp" $gp "comp" $gp.components.workspace.desktopIdeImages.codeDesktop)) }}
-  code-desktop-insiders:
-    orderKey: "03"
-    title: "VS Code"
-    type: "desktop"
-    logo: "https://ide.{{ $.Values.hostname }}/image/ide-logo/vscodeInsiders.svg"
-    tooltip: "Visual Studio Code Insiders for early adopters."
-    label: "Insiders"
-    image: {{ (include "gitpod.comp.imageFull" (dict "root" $ "gp" $gp "comp" $gp.components.workspace.desktopIdeImages.codeDesktopInsiders)) }}
+    latestImage: {{ (include "gitpod.comp.imageFull" (dict "root" $ "gp" $gp "comp" $gp.components.workspace.desktopIdeImages.codeDesktopInsiders)) }}
   intellij:
     orderKey: "04"
     title: "IntelliJ IDEA"
```

Comment:
It can like this 👍, in this case, we need to determine the details of the text @loujaybee

Reply:
I'd vote for going with the suggestion in https://github.com/gitpod-io/gitpod/pull/8783#discussion_r828511377, otherwise, let's open a follow up issue for this to discuss further. 🏓

Reply:
**fyi**: Opened follow up issue in https://github.com/gitpod-io/gitpod/issues/9201 to track this discussion.